src/DETHRACE/common/displays.c

```diff
@@ -1162,24 +1163,27 @@ void DoInstruments(tU32 pThe_time) {
             }
 #if defined(DETHRACE_FIX_BUGS)
 /*
- * The OG derives gear mask height of 28 by `gears_image->height / 8`, but
+ * The OG derives gear mask height of 16 or 28 by `gears_image->height / 8`, but
  * this is only valid for HGEARS.PIX, which contains 8 gear images. Hardcoding
  * this number fixes gear rendering for cars using HGEARS4.PIX, which consists
  * of 11 gear images.
  */
-#define GEAR_HEIGHT 28
+#define GEAR_HEIGHT 16
+#define GEAR_HEIGHT_HIRES 28
 #else
 #define GEAR_HEIGHT ((int)gProgram_state.current_car.gears_image->height / 8)
+#define GEAR_HEIGHT_HIRES GEAR_HEIGHT
```

Comment:
did you mean to include this line?

Reply:
I did. For OG behavior, we need the mask in both resolutions to be derived from the `(int)gProgram_state.current_car.gears_image->height / 8)` calculation.

Reply:
Ah, of course, I missed the #else